yt_dlp/extractor/hypergryph.py

```diff
@@ -5,28 +5,24 @@
 class MonsterSirenHypergryphMusicIE(InfoExtractor):
     _VALID_URL = r'https?://monster-siren\.hypergryph\.com/music/(?P<id>\d+)'
     _TESTS = [{
-        'url': 'https://monster-siren.hypergryph.com/music/514562',
+        'url': 'https://monster-siren.hypergryph.com/music/779464',
         'info_dict': {
-            'id': '514562',
+            'id': '779464',
             'ext': 'wav',
             'artists': ['塞壬唱片-MSR'],
-            'album': 'Flame Shadow',
-            'title': 'Flame Shadow',
+            'title': '无机物',
```

Comment:
Don't change the test URL if the link isn't dead. The point of these tests is to make sure we can get the same results after we make changes to the extractor. There was no point to this test whatsoever if you completely change the URL at the same time you make a change to the extractor code